Radius(x).py

```diff
@@ -6,31 +6,68 @@
 @author: amelielaurens
 """
 
+# Predict the radius of the jet in steady state as a function of the axial coordinate x
+
 from models_RJS import *
 import numpy
 import matplotlib.pyplot as plt
 
-# predict the radius of the jet in steady state as a function of the axial coordinate x
-surface_tension =float(input("Enter the surface tension in kg/s^2 : "))
-Rc = float(input("Enter the radius of the collector in m : "))
-s0 = float(input("Enter the radius of the reservoir in m : "))
-discretisation = int(input("Enter an int for the mesh's thinness : "))
+
+discretisation = 20 #The higher the discretisation number is, the finer the discretisation will be, there will be more points on the graphic
+
+
+#Choose one machine and one polymer
+#Machines
+#Super Floss Maxx
+#s0 = 0.06985
+#Rc = 0.3302
+#orifice_radius = 0.001512
+#omega = 57.5
+
+#CANDY-V001
+s0 = 0.0635
+Rc = 0.254
+orifice_radius = 0.000267
+omega = 40.
+
+
+#Polymers
+#PP
+#surface_tension = 0.0436
+#rho = 900.
+#mu = 0.63
+
+#PLA
+surface_tension = 0.0248
+rho = 1250.
+mu = 0.113
+
+
+#If we want the user to enter its own values
+#surface_tension =float(input("Enter the surface tension in kg/s^2 : "))
+#Rc = float(input("Enter the radius of the collector in m : "))
+#s0 = float(input("Enter the radius of the reservoir in m : "))
+#discretisation = int(input("Enter an int for the mesh's thinness : "))
+#orifice_radius = float(input("Enter the radius of the orifice in m : "))
+#rho = float(input("Enter the density of the polymer in kg/m^3 : "))
+#mu = float(input("Enter the viscosity of the polymer in Pa.s : "))
+#omega = float(input("Enter the angular velocity in rounds per second : "))
+
+
 x_position = numpy.linspace(0, Rc-s0, discretisation)
-r0 = float(input("Enter the radius of the orifice in m : "))
-rho = float(input("Enter the density of the polymer in kg/m^3 : "))
-omega_th=critical_rotational_velocity_threshold(surface_tension, r0, s0, rho)
```

Comment:
Add a small comment at the end of the line when you introduce the variable to let the reader know what it is.

Reply:
I think that @AmelieLaurens you were using docstrings (https://www.python.org/dev/peps/pep-0257/) up to know to explain how functions work, that is good, you should keep using that.